overlord/restart/restart.go

```diff
@@ -274,10 +272,13 @@ func notifyRebootRequiredClassic(task *state.Task) error {
 	// as apt/dpkg.
 	nrrPath := filepath.Join(dirs.GlobalRootDir, "/usr/share/update-notifier/notify-reboot-required")
 	if osutil.FileExists(nrrPath) {
-		cmd := exec.Command(nrrPath)
-		cmd.Env = os.Environ()
-		cmd.Env = append(cmd.Env, fmt.Sprintf("DPKG_MAINTSCRIPT_PACKAGE=snap:%s", rebootRequiredSnap))
-		cmd.Env = append(cmd.Env, "DPKG_MAINTSCRIPT_NAME=postinst")
+		snapStr := fmt.Sprintf("snap:%s", rebootRequiredSnap)
```

Comment:
you decided to go with snap:snapd instead of just snapd for the fallback case?

Reply:
Sorry, I misunderstood the original proposal I guess. I have no strong opinion either way, but just `snapd` seems indeed cleaner if there is no further infomration.